manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -84,18 +84,25 @@ spec:
             runAsGroup: 0
             runAsUser: 0
             runAsNonRoot: false
-{{- if .Values.cni.ambient.enabled }}
+            # Both ambient and sidecar repair mode require elevated node privileges to function.
+{{- if or .Values.cni.ambient.enabled .Values.cni.repair.enabled }}
+            # privileged is redundant with CAP_SYS_ADMIN
+            # since it's redundant, hardcode it to `true`, then manually drop ALL + readd granular
+            # capabilities we actually require
+            privileged: true
```

Comment:
why still need to set priviledged with the below capabilities added
 CAP_SYS_ADMIN
CAP_NET_ADMIN
CAP_NET_RAW

Reply:
@hzxuzhonghu I don't have an OpenShift env yet to test with, so I can't rule out the possibility that OpenShift might require `privileged: true` to be set before it will allow a container to drop all capabilities and re-add granular ones.

Also, I think it's preferable in the short term to have both for compat reasons - if anyone has any, say, existing pod validation hooks that are checking for `privileged: true` but not checking `securityContext`, setting both means the pod would still get flagged as requiring elevated permissions.

Setting this to true here in this conditional basically means the templated pod manifest will be backwards compatible for all cases where `privileged` would have been set to true before.

Reply:
make sense